src/libs/Navigation/Navigation.js

```diff
@@ -73,7 +73,7 @@ const getActiveRouteIndex = function (route, index) {
 /**
  * Main navigation method for redirecting to a route.
  * @param {String} route
- * @param {String} type - Type of action to perform. Currently UP is supported.
+ * @param {String} [type] - Type of action to perform. Currently UP is supported.
```

Comment:
you can remove those JSdoc params , but you can leave descriptions

Reply:
@kubabutkiewicz This is a JS file, he just change the type to avoid errors in his TS file.